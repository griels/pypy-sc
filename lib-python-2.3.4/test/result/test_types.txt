command: /usr/bin/python /u/pedronis/PyPy/dist/pypy/tool/alarm.py 900 /u/pedronis/PyPy/dist/pypy/interpreter/py.py --oldstyle /u/pedronis/PyPy/dist/pypy/lib/test2/test_types.py
run by: pedronis@ratthing-b246
sys.platform: linux2
sys.version_info: (2, 3, 5, 'final', 0)
startdate: Thu Apr 28 14:54:18 2005
pypy-revision: 11549
OUTPUT TEST
see output in: /u/pedronis/PyPy/dist/lib-python-2.3.4/test/result/test_types.out
============================================================
faking <type 'unicode'>
faking <type 'module'>
faking <type 'file'>
faking <type 'posix.stat_result'>
faking <type 'posix.statvfs_result'>
FAILED: test output differs
========================== closed ==========================
execution time: 10.963078022 seconds
exit status: 0
