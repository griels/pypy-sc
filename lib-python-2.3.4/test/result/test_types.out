test_types
6. Built-in types
6.1 Truth value testing
6.2 Boolean operations
6.3 Comparisons
6.3.1 Conversion errors
6.3.2 Division errors
6.4 Numeric types (mostly conversions)
6.4.1 32-bit integers
6.4.2 Long integers
6.4.3 Floating point numbers
6.5 Sequence types
6.5.1 Strings
6.5.2 Tuples
6.5.3 Lists
6.5.3a Additional list operations
6.6 Mappings == Dictionaries
6.6.1 dict methods
6.6.2 user-dict methods
6.6.3 dict.fromkeys
6.6.4 dict copy, get, setdefault
6.6.5 dict popitem
6.6.6 dict pop
6.7 type
6.99999999...   All tests ran to completion
